### PR TITLE
docs: add design accessibility guidelines

### PR DESCRIPTION
**Pull request by leeejune**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Fixes:** #155

## Summary:

Added design accessibility notes.

## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [ ] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [x] add docsumentation

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: docs/accessibility.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+## Text
+
+`auro-button` uses the standard body size text of 16px. 
+
+## Color
+The default blue color (`auro-color-ui-default-on-light`) has a contrast ratio of *4.84:1* against white. The color contrast tests pass for AA and AAA levels except for the AAA level on normal text.
+
+### References
+[WCAG 2.1 1.4.3] The visual presentation of text and images of text has a contrast ratio of at least *4.5:1*.
```
Comment (blackfalcon) [inline docs/accessibility.md:11]: Do we have links to the WCAG specs?

File: docs/accessibility.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+[WCAG 2.1 1.4.1] Color is not used as the only visual means of conveying information.
+
+## Interactions 
+
+### Tertiary Button
+Although the tertiary button does not have an underline by default, the hover state has an underline. According to WCAG 2.1 1.4.1, color can be used as the only indicator of a link if “the contrast ratio between the link and the surrounding text is at least 3:1 and an additional distinction (e.g., it becomes underlined) is provided when the link is hovered over and receives focus.” User research also indicates that color blind users can differentiate between an underlined tertiary button and normal text. In this case, `auro-button` is differentiated from a hyperlink and adheres to accessibility guidelines.
```
Comment (blackfalcon) [inline docs/accessibility.md:18]: This is good stuff. 

Is there something we can link to for [WCAG 2.1 1.4.1]?

**Review by blackfalcon — CHANGES_REQUESTED** [review]

File: docs/accessibility.md (hunk 2 above)
Comment (leeejune) [reply docs/accessibility.md]: Great idea. Added the links!

**Review by blackfalcon — APPROVED** [review]

**Pull request merged**